lib/fig/figrc.rb

```diff
@@ -28,17 +28,44 @@ def self.find(
 
     handle_override_configuration(configuration, override_path)
     handle_figrc(configuration) if not disable_figrc
+    
+    # Check for legacy environment variable usage
+    consume_url = derive_repository_url(specified_repository_url, 'CONSUME', configuration)
```

Comment:
I think we should probably consider renaming this so that the ruby variables are consistent with the environment variables.

```suggestion
    consume_url = derive_repository_url(consume_repository_url, 'CONSUME', configuration)
```

Reply:
fixed in d3a400c